include/zephyr/bluetooth/services/hrs.h

```diff
@@ -19,10 +19,50 @@
 
 #include <stdint.h>
 
+#include <zephyr/sys/slist.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/** @brief Heart rate service callback structure */
+struct bt_hrs_cb {
+	/** @brief Heart rate notifications changed
+	 *
+	 * @param enabled Flag that is true if notifications were enabled, false
+	 *                if they were disabled.
+	 */
+	void (*ntf_changed)(bool enabled);
+
+	/** Internal member to form a list of callbacks */
+	sys_snode_t node;
```

Comment:
```suggestion
	sys_snode_t _node;
```

It's common practice to prefix internal fields with `_`

Reply:
Updated